lib/tasks/test.js

```diff
@@ -146,14 +147,14 @@ module.exports._runBrowserTest = function(config) {
 			configContent = JSON.parse(configContent);
 			configContent.test_settings.default.launch_url = config.testUrl;
 
-			fs.writeFile(nightwatchConfigPath, JSON.stringify(configContent), function(error) {
+			fs.writeFile(nightwatchConfigPath.replace('json', 'tmp.json'), JSON.stringify(configContent), 'utf8', function(error) {
```

Comment:
Makes sense, I just mean de-duplicate the logic on line 150 and 157 so you only do this in one place.